src/components/EmojiPicker/EmojiPickerMenu/emojiPickerMenuPropTypes.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import CONST from '@src/CONST';
+
+const emojiPickerMenuPropTypes = {
+    /** Function to add the selected emoji to the main compose text input */
+    onEmojiSelected: PropTypes.func.isRequired,
+
+    /** Stores user's preferred skin tone */
+    preferredSkinTone: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
Looks like, thanks to the hooks you added, the only prop still needed for `EmojiPickerMenu` is `onEmojiSelected`. So let's remove these other two along with their defaults